src/index.css

```diff
@@ -299,12 +302,6 @@ html:is(.dark) [data-amplify-authenticator] {
     @apply field-input;
   }
 
-  /* This css is required to handle createElement error UI  */
-  .field > textarea[aria-invalid="true"],
-  .field > input[aria-invalid="true"] {
-    @apply border-red;
```

Comment:
`field`'s border should not change based on child input's state -> only the child input's border should change, which is handled with `field-input` styling (which is by extension applied automatically to all `.field > textarea, .field > input` components, see above)